tests/test_op12_schema_vs_schema_validation.py

```
@@ -3570,5 +3570,150 @@ def test_start(self):
+                "properties": {
+                    "extra": {"type": "string"},
+                },
+            },
+            "register derived from final base",
+        ),
+        _validate_schema(
+            "gts.x.test12.final.base.v1~x.test12._.derived.v1~",
+            False,
+            "validate derived from final base should fail",
+        ),
+    ]
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)